### PR TITLE
feat: use new osx common network configs

### PR DESCRIPTION
**Pull request by clauBv23**
Refactor to use the new commons config networks. 

[OS-1109](https://aragonassociation.atlassian.net/browse/OS-1109)

[OS-1109]: https://aragonassociation.atlassian.net/browse/OS-1109?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,12 @@
 
 ## Quickstart
 
-After [creating a new repository from this template](https://github.com/new?template_name=osx-plugin-template-hardhat&template_owner=aragon), cloning, and opening it in your IDE, run
+After [creating a new repository from this template](https://github.com/new?template_name=osx-plugin-template-hardhat&template_owner=aragon), cloning, and opening it in your IDE, create a `.env` file from the `.env.example` file and put in the Alchemy API key, feel free to add other API keys for the services that you want to use. Now run,
```
Comment (heueristik) [inline README.md]:
```suggestion
After [creating a new repository from this template](https://github.com/new?template_name=osx-plugin-template-hardhat&template_owner=aragon), cloning, and opening it in your IDE, create an `.env` file from the `.env.example` file and put in the Alchemy API key. Feel free to add other API keys for the services that you want to use. Now run,
```

File: .env.example [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+NETWORK_NAME="sepolia" #  ["mainnet", "sepolia", "polygon", "mumbai","baseMainnet", "baseGoerli", "baseSepolia", "arbitrum", "arbitrumSepolia"]
 
 # CONTRACTS
 
 ## One or multiple hex encoded private keys separated by commas `,` replacing the hardhat default accounts.
 PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" # Default hardhat account 0 private key. DON'T USE FOR DEPLOYMENTS
 
-## Infura credentials
-INFURA_API_KEY="zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+## RPC alchemy credentials
```
Comment (heueristik) [inline .env.example]:
```suggestion
## Alchemy RPC endpoint credentials
```

**Review by jordaniza — DISMISSED** [review]

**Review by heueristik — CHANGES_REQUESTED** [review]
The workflows fail becasue you haven't changed `INFURA_API_KEY` to `ALCHEMT_API_KEY` yet (see https://github.com/aragon/osx-plugin-template-hardhat/actions/runs/8245959072).

The github secret is already set
Bildschirmfoto 2024-03-12 um 09 49 30

**Review by heueristik — CHANGES_REQUESTED** [review]
Please bump the osx-commons-configs also in `packages/subgraph`

**Comment by clauBv23** [comment]
> Please bump the osx-commons-configs also in `packages/subgraph`
@Michael-A-Heuer should I also bump the versions in commons-sdk?

**Comment by heueristik** [comment]
> > Please bump the osx-commons-configs also in `packages/subgraph`
> > @Michael-A-Heuer should I also bump the versions in commons-sdk?

Yes, every repo should use the latest version of our packages (at least on `develop`).

**Review by heueristik — APPROVED** [review]
Ok to be merged, although the subgraph tests are failing. The latter will be fixed in [OS-1028](https://aragonassociation.atlassian.net/browse/OS-1028).

**Comment by clauBv23** [comment]
> Ok to be merged, although the subgraph tests are failing. The latter will be fixed in [OS-1028](https://aragonassociation.atlassian.net/browse/OS-1028).

contracts tests are also failing (due to the ens being taken on sepolia), just not being shown on the actions because in the last commit, contracts were not changed  @Michael-A-Heuer 

[OS-1028]: https://aragonassociation.atlassian.net/browse/OS-1028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Review by jordaniza — APPROVED** [review]

**Pull request merged**